css/properties/all.json

```diff
@@ -9,10 +9,10 @@
               "version_added": "37"
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": "37"
```

Comment:
This also applies to desktop and Android webview. It's also in Opera 24. The Chrome Status entry doesn't list webview, but I've confirmed it. 

https://www.chromestatus.com/features/6178222542684160

Reply:
Done